precode.go

```diff
@@ -40,13 +41,81 @@ var tasks = map[string]Task{
 }
 
 // Ниже напишите обработчики для каждого эндпоинта
-// ...
+func getAllTasks(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var taskList []Task
+	for _, task := range tasks {
+		taskList = append(taskList, task)
+	}
+
+	if err := json.NewEncoder(w).Encode(taskList); err != nil {
```

Comment:
Можно не делать промежуточный массив

```suggestion
	if err := json.NewEncoder(w).Encode(tasks); err != nil {
```